packages/grid/_modules_/grid/components/cell/GridRowCells.tsx

```diff
@@ -131,4 +132,88 @@ export const GridRowCells = React.memo(function GridRowCells(props: RowCellsProp
       ))}
     </React.Fragment>
   );
-});
+}
+
+const GridRowCells = React.memo(GridRowCellsRaw);
+
+GridRowCellsRaw.propTypes = {
+  // ----------------------------- Warning --------------------------------
+  // | These PropTypes are generated from the TypeScript type definitions |
+  // | To update them edit the TypeScript types and run "yarn proptypes"  |
+  // ----------------------------------------------------------------------
+  cellClassName: PropTypes.string,
+  cellFocus: PropTypes.shape({
+    field: PropTypes.string.isRequired,
+    id: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
+  }),
+  cellTabIndex: PropTypes.shape({
+    field: PropTypes.string.isRequired,
+    id: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
+  }),
+  columns: PropTypes.arrayOf(
```

Comment:
Do we want to have the details for this type ? It's quite heavy
Same thing for the other `columns`, `currentColumn` or `colDef`